package.json

```diff
@@ -6,12 +6,13 @@
   "scripts": {
     "test": "npx hardhat test",
     "build": "hardhat compile",
+    "clean": "npx hardhat clean",
     "console": "hardhat console",
-    "verify": "yarn hint && yarn test",
     "lint": "yarn solhint 'contracts/**/*.sol' && yarn prettier -c './**/*.js'",
     "format": "yarn prettier --write contracts/*.sol && yarn prettier --write test/*.js",
     "hint": "solhint \"contracts/**/*.sol\"",
-    "coverage": "yarn hardhat coverage  --testfiles \"test/*.js\""
+    "coverage": "yarn hardhat coverage  --testfiles \"test/*.js\"",
+    "deploy": "npx hardhat run scripts/deploy.js"
```

Comment:
Add a new command i.e. `yarn upgrade-checks` that will specifically run this script.